spec/integration/import_entry_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
+require "flex_commerce_api"
+require "uri"
+
+RSpec.describe FlexCommerce::ImportEntry do
+  # Global context for all specs - defines things you dont see defined in here
+  # such as flex_root_url, api_root, default_headers and page_size
+  # see api_globals.rb in spec/support for the source code
+  include_context "global context"
+  let(:subject_class) { ::FlexCommerce::ImportEntry }
+
+  context "with fixture files from flex" do
+    context "working with a single import entry" do
+      let(:singular_resource) { build(:import_entry_from_fixture) }
+      before :each do
+        stub_request(:get, "#{api_root}/imports/1/import_entries/1.json_api").with(headers: { "Accept" => "application/vnd.api+json" }).to_return body: singular_resource.to_h.to_json, status: response_status, headers: default_headers
+      end
+      subject { subject_class.where(import_id: 1).find(1).first }
+      it_should_behave_like "a singular resource with an error response"
```

Comment:
Very nit picky but could you add some whitespace around the various `context` / `it` blocks please and in the other spec below, it makes it much easier to read IMO